src/deploy/releases_store.py

```diff
@@ -5,10 +5,10 @@
 
 
 class DynamoDbReleaseStore:
-    def __init__(self, project_id, role_arn=None):
+    def __init__(self, project_id, region_name, role_arn=None):
         self.project_id = project_id
         self.table_name = f"wellcome-releases-{project_id}"
-        self.session = Iam.get_session("ReleaseToolDynamoDbReleaseStore", role_arn)
+        self.session = Iam.get_session("ReleaseToolDynamoDbReleaseStore", region_name, role_arn)
```

Comment:
Maybe sprinkle some keyword arguments around so functions are a bit more resilient to signature changes like this?

Reply:
That wasn't what caused the issue here (see https://github.com/wellcomecollection/weco-deploy/pull/18/files#diff-91466e5dd6202aef5752a56d28bb62a7L162) - i've just pushed region_name everywhere in this PR so it has no default value.

Reply:
But in general 👍 to keyword arguments everywhere.